website/templates/project/private_links.mako

```diff
@@ -0,0 +1,24 @@
+<%include file="modal_generate_private_link.mako"/>
+<link rel="stylesheet" href="/static/css/private_links_table.css">
+<div class="scripted" id="linkScope">
+    <table id="privateLinkTable" class="table responsive-table responsive-table-xs"
```

Comment:
<img width="978" alt="screen shot 2017-06-15 at 10 36 01 am" src="https://user-images.githubusercontent.com/1322421/27186775-568ef682-51b7-11e7-875a-0c69d70d8806.png">
The table is not adequately responsive on the node settings page. Works fine on the contrib page.